.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file can be removed as i've got this currently on my local branch, so will cause merge conflict! :D

Reply:
Ok done